docs/.eslintrc.cjs

```diff
@@ -69,6 +69,8 @@ module.exports = {
 			files: ["docusaurus.config.ts", "playwright.config.ts", "infra/**/*"],
 			rules: {
 				"import/no-internal-modules": "off",
+				"import/no-nodejs-modules": "off",
+				"unicorn/no-process-exit": "off",
```

Comment:
Nit: I'd make the no-process-exit a local disable instead of a global one.

Reply:
Ah, no, leave it there then. I only saw the one in the PR but if there are others it's fine to keep here. If anything, they might be doing local disables right now and they can be removed?

Reply:
Well, we don't actually run the linter against `.js` code yet. That's something I need to work on. Our base config is designed for `.ts`, and yields _many_ errors when run against `.js`. But intellisense complains, and this rule isn't TS-specific, so I figured I'd just tackle it now.